reporters/GlobalSummaryReporter.js

```diff
@@ -0,0 +1,476 @@
+    };
+
+    // - Cumulative collateral withdrawn from contract
+    let collateralWithdrawn = toBN("0");
+    let collateralWithdrawnPeriod = toBN("0");
+    for (let event of this.collateralWithdrawEvents) {
+      collateralWithdrawn = collateralWithdrawn.add(toBN(event.returnValues.value));
+      if (event.blockNumber >= this.startBlockNumberForPeriod) {
+        collateralWithdrawnPeriod = collateralWithdrawnPeriod.add(toBN(event.returnValues.value));
+      }
+    }
+    allSponsorStatsTable["collateral withdrawn"] = {
+      cumulative: this.formatDecimalString(collateralWithdrawn),
+      [this.periodLabelInHours]: this.formatDecimalString(collateralWithdrawnPeriod)
+    };
+
+    // - Net collateral deposited into contract:
+    let netCollateralWithdrawn = collateralDeposited.sub(collateralWithdrawn);
+    let netCollateralWithdrawnPeriod = collateralDepositedPeriod.sub(collateralWithdrawnPeriod);
+    allSponsorStatsTable["net collateral deposited"] = {
+      cumulative: this.formatDecimalString(netCollateralWithdrawn),
+      [this.periodLabelInHours]: this.formatDecimalString(netCollateralWithdrawnPeriod)
+    };
+
+    // - Tokens minted: tracked via Create events.
+    let tokensMinted = toBN("0");
+    let tokensMintedPeriod = toBN("0");
+    for (let event of this.createEvents) {
+      tokensMinted = tokensMinted.add(toBN(event.tokenAmount));
+      if (event.blockNumber >= this.startBlockNumberForPeriod) {
+        tokensMintedPeriod = tokensMintedPeriod.add(toBN(event.tokenAmount));
+      }
+    }
+    allSponsorStatsTable["tokens minted"] = {
+      cumulative: this.formatDecimalString(tokensMinted),
+      [this.periodLabelInHours]: this.formatDecimalString(tokensMintedPeriod),
+      current: this.formatDecimalString(this.totalTokensOutstanding)
+    };
+
+    // - Tokens burned
+    let tokensBurned = toBN("0");
+    let tokensBurnedPeriod = toBN("0");
+    for (let event of this.syntheticBurnedEvents) {
+      tokensBurned = tokensBurned.add(toBN(event.returnValues.value));
+      if (event.blockNumber >= this.startBlockNumberForPeriod) {
+        tokensBurnedPeriod = tokensBurnedPeriod.add(toBN(event.returnValues.value));
+      }
+    }
+    allSponsorStatsTable["tokens burned"] = {
+      cumulative: this.formatDecimalString(tokensBurned),
+      [this.periodLabelInHours]: this.formatDecimalString(tokensBurnedPeriod)
+    };
+
+    // - Net tokens minted:
+    let netTokensMinted = tokensMinted.sub(tokensBurned);
+    let netTokensMintedPeriod = tokensMintedPeriod.sub(tokensBurnedPeriod);
+    allSponsorStatsTable["net tokens minted"] = {
+      cumulative: this.formatDecimalString(netTokensMinted),
+      [this.periodLabelInHours]: this.formatDecimalString(netTokensMintedPeriod)
+    };
+
+    // - GCR (collateral / tokens outstanding):
+    let currentCollateral = toBN(this.totalPositionCollateral.toString());
+    let currentTokensOutstanding = toBN(this.totalTokensOutstanding.toString());
```

Comment:
nit: are these already BNs or strings?

Reply:
Nope see above

Reply:
- raw value: [ '75166285203707845817894', rawValue: '75166285203707845817894' ]
- toString'd: 75166285203707845817894 
- typeof: object

Reply:
This is so weird.

Reply:
I've noticed this for a while

Reply:
Maybe solidity types have some sort of `.toString()` inherent method